charts/argo-cd/templates/argocd-application-controller/clusterrole.yaml

```diff
@@ -1,15 +1,16 @@
-{{- if .Values.clusterAdminAccess.enabled }}
+{{- if .Values.controller.enabled }}
```

Comment:
I have doubts that this is correct, there are 2 ways to configure argo-cd, if cluster admin is enabled it means it will be able to install application also in the cluster where it runs, if it is disabled then it most likely install the applications in external clusters (so it doesn't need cluster wide rights). So I don't see any connection with enabling/disabling the controller.

Reply:
This is moved to the controller block and updated readme/comments https://github.com/argoproj/argo-helm/pull/129/commits/e2215f864ab833364e2c294f63d86755d0e05186